templates/civimelder/mijn_activiteiten_lijst.html.twig

```diff
@@ -0,0 +1,67 @@
+<tr id="civimelder-row-{{ activiteit.getId }}">
+	<td>
+		<div class="Activiteit">
+			{{ activiteit.getTitel() }}
+		</div>
+	</td>
+	<td>
+		<div class="Van">
+			{{ activiteit.getStart() | datetime_format_long }} {{ activiteit.getStart() | time_format }}
+		</div>
+	</td>
+	<td>
+		<div class="Tot">
+			{{ activiteit.getEinde() | datetime_format_long }} {{ activiteit.getEinde() | time_format }}
+		</div>
+	</td>
+	<td>
+		<div class="Aanwezigen (limiet)">
+			{{ activiteit.getAantalAanmeldingen() }}  ({{ activiteit.getCapaciteit() }})
```

Comment:
Je hoeft hier toch niet expliciet de getters te gebruiken? alsin `activiteit.aantalAanmeldingen` geeft toch hetzelfde resultaat.

Reply:
In dit geval wel ja. Voor de andere dingen wil ik wel de getter gebruiken, omdat mijn getter specifieke logica heeft m.b.t. de inheritance tussen activiteiten en reeksen.

De priority van Twig ligt bij directe properties, daarna pas getters (https://twig.symfony.com/doc/3.x/templates.html)
> For convenience's sake foo.bar does the following things on the PHP layer:
    - check if foo is an array and bar a valid element;
    - if not, and if foo is an object, check that bar is a valid property;
    - if not, and if foo is an object, check that bar is a valid method (even if bar is the constructor - use __construct() instead);
    - if not, and if foo is an object, check that getBar is a valid method;
    - if not, and if foo is an object, check that isBar is a valid method;
    - if not, and if foo is an object, check that hasBar is a valid method;
    - if not, return a null value.